### PR TITLE
Evaluate instruction dataset

### PR DESCRIPTION
**Pull request by binkjakub**

### CODE REVIEW

File: data/results/pl_court_instruct/zero_shot/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/results_llama_3_8B.json
+/results_meta_llama_3_8B_instruct.json
```
Comment (asawczyn) [inline data/results/pl_court_instruct/zero_shot/.gitignore]: do we need both?

File: scripts/sft/evaluate_zero_shot.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        max_length=max_length,
+        enable_padding=enable_padding,
+    )
+    encoded_ds = ds["test"].map(
+        encoder,
+        batched=False,
+        num_proc=10,
+    )
+    encoded_ds.set_format("torch")
+    encoded_ds = encoded_ds.select(range(20))
```
Comment (asawczyn) [inline scripts/sft/evaluate_zero_shot.py]:
```suggestion
```

**Comment by asawczyn** [comment]
+ conflicts & CI checks

File: data/results/pl_court_instruct/zero_shot/.gitignore (hunk 1 above)
Comment (binkjakub) [reply data/results/pl_court_instruct/zero_shot/.gitignore]: first to be deleted

**Pull request merged**